content/en/blog/2025/sail-operator-ga/index.md

```diff
@@ -0,0 +1,301 @@
+---
+title: "Sail operator 1.0.0 has been released: manage Istio with an operator"
+description: Sail Operator 1.0.0 released, a project part of the istio-ecosystem organization. Let's dive in into the basics of the Sail Operator and let's run an example to show how easy it is to manage Istio with it.
+publishdate: 2025-03-19
+attribution: "Francisco Herrera - Red Hat"
+keywords: [istio,operator,sail,incluster,istiooperator]
+---
+
+The Sail operator is GA with a clear mission: to simplify and streamline Istio management in your cluster. By using the Kubernetes operator pattern, Sail operator aims to add new value to istio by improving the upgrade process.
```

Comment:
Probably want to start with an overview/recap of what the Sail Operator is and a link back to https://istio.io/latest/blog/2024/introducing-sail-operator/

```suggestion
The [Sail Operator](https://github.com/istio-ecosystem/sail-operator) is a community project launched by Red Hat to build a modern [operator](https://www.redhat.com/en/topics/containers/what-is-a-kubernetes-operator) for Istio. [First announced in August 2024](https://github.com/istio-ecosystem/sail-operator) we are pleased to announce Sail Operator is now GA with a clear mission: to simplify and streamline Istio management in your cluster..
```